runestone/mchoice/multiplechoice.py

```diff
@@ -210,12 +249,15 @@ def run(self):
             """
         addQuestionToDB(self)
 
-        mcNode = MChoiceNode(self.options, rawsource=self.block_text)
-        mcNode.source, mcNode.line = self.state_machine.get_source_and_line(self.lineno)
-        mcNode.template_start = TEMPLATE_START
-        mcNode.template_option = OPTION
-        mcNode.template_end = TEMPLATE_END
-
+        mcNode = MChoiceNode()
+        # This is maybe a hack or maybe the new right way to do this...
+        mcNode["runestone_options"] = self.options
+        mcNode["source"], mcNode["line"] = self.state_machine.get_source_and_line(
+            self.lineno)
+        mcNode["template_start"] = TEMPLATE_START
```

Comment:
Why assign constants to each instance of `mcNode`? This would make sense as globals in this module instead.